CovidWatch/libcontactrace/src/main/java/org/covidwatch/libcontactrace/BluetoothService.kt

```diff
@@ -0,0 +1,104 @@
+package org.covidwatch.libcontactrace
+
+import android.app.Notification
+import android.bluetooth.BluetoothAdapter
+import android.content.Intent
+import android.os.Binder
+import android.os.IBinder
+import androidx.lifecycle.LifecycleService
+import org.covidwatch.libcontactrace.cen.CenGenerator
+import org.covidwatch.libcontactrace.cen.CenVisitor
+import java.util.*
+
+class BluetoothService : LifecycleService() {
```

Comment:
one pretty big difference from the previous `libcontacttracing` is that this service is part of the library, right?

Reply:
That's right.
This way a client of the library just need to add to gradle and start using out of the box.